### PR TITLE
v1.6.0 release notes fixes

### PR DESCRIPTION
**Pull request by xaur**

### CODE REVIEW

**Comment by xaur** [comment]
missing dcrd chunk restored from https://github.com/decred/dcrd/blob/master/docs/release-notes/release-notes-1.6.0.md

**Comment by xaur** [comment]
Using small commits for easier review.

File: release-notes.md [hunk 1]
```diff
@@ -1254,11 +1254,11 @@ follow what dcrwallet is doing, there is a log window below.  In the future,
 we may add better messaging that will allow updates to the mixing process 
 instead of just showing raw logs.
 
-Once privacy is enabled we have restricted external spending to the mixed 
-account.  This is to ensure privacy is not broken by spending from any unmixed 
-account.  Additionally, we have restricted address generation to the mixed 
-account, since only funds that have been properly mixed should be allowed to 
-end up there.
+Once privacy is enabled spending to external addresses is only allowed from the
```
Comment (matthawkins90) [inline release-notes.md]: Consider adding a comma for clarity: "Once privacy is enabled, spending to..."

**Comment by matthawkins90** [comment]
Otherwise, good updates!

File: release-notes.md (hunk 1 above)
Comment (xaur) [reply release-notes.md]: Added.

File: release-notes.md [hunk 2]
```diff
@@ -1263,53 +1457,53 @@ Please read the [initial release (v0.1.0) notes](https://github.com/decred/dcrde
 
 - The account ID for each configured DEX server is now displayed on the settings 
 page.  When not logged in, there is a placeholder that says to login to display 
-the account ID. (https://github.com/decred/dcrdex/commit/1a5c070196ab7899214b524a9c681168fbdcfd75)
-- Focus password field in order dialogs. (https://github.com/decred/dcrdex/commit/5eb9fb2de51722158eaf1d2122c11f30154bd9b3)
-- Colorize the "Side" column in the orders table. (https://github.com/decred/dcrdex/commit/83b07cd08a8a7ecced012335092c0f196d7fcfb0)
+the account ID. ([1a5c070](https://github.com/decred/dcrdex/commit/1a5c070196ab7899214b524a9c681168fbdcfd75))
+- Focus password field in order dialogs. ([5eb9fb2](https://github.com/decred/dcrdex/commit/5eb9fb2de51722158eaf1d2122c11f30154bd9b3))
+- Colorize the "Side" column in the orders table. ([83b07cd](https://github.com/decred/dcrdex/commit/83b07cd08a8a7ecced012335092c0f196d7fcfb0))
 - The registration fee address is no longer logged if their is a funding error
```
Comment (matthawkins90) [inline release-notes.md]: (Grammar) *there

File: release-notes.md [hunk 3]
```diff
@@ -1263,53 +1457,53 @@ Please read the [initial release (v0.1.0) notes](https://github.com/decred/dcrde
...
 - The account ID for each configured DEX server is now displayed on the settings 
 page.  When not logged in, there is a placeholder that says to login to display 
-the account ID. (https://github.com/decred/dcrdex/commit/1a5c070196ab7899214b524a9c681168fbdcfd75)
-- Focus password field in order dialogs. (https://github.com/decred/dcrdex/commit/5eb9fb2de51722158eaf1d2122c11f30154bd9b3)
-- Colorize the "Side" column in the orders table. (https://github.com/decred/dcrdex/commit/83b07cd08a8a7ecced012335092c0f196d7fcfb0)
+the account ID. ([1a5c070](https://github.com/decred/dcrdex/commit/1a5c070196ab7899214b524a9c681168fbdcfd75))
+- Focus password field in order dialogs. ([5eb9fb2](https://github.com/decred/dcrdex/commit/5eb9fb2de51722158eaf1d2122c11f30154bd9b3))
+- Colorize the "Side" column in the orders table. ([83b07cd](https://github.com/decred/dcrdex/commit/83b07cd08a8a7ecced012335092c0f196d7fcfb0))
 - The registration fee address is no longer logged if their is a funding error 
 since there is nothing a user can do with the address other than shoot their
```
Comment (matthawkins90) [inline release-notes.md]: (Grammar) *themselves in the foot

File: release-notes.md (hunk 2 above)
Comment (xaur) [reply release-notes.md]: Nice catch, fixed.

File: release-notes.md (hunk 3 above)
Comment (xaur) [reply release-notes.md]: Haha. Fixed.

**Comment by xaur** [comment]
@alexlyp @jrick I added Code Contributors for dcrwallet and decrediton to match other projects but this can be reverted if it's not something you wanted.

**Review by alexlyp — APPROVED** [review]

**Pull request merged**